<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataInfo</class>
 <widget class="QMainWindow" name="dataInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataInfo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../PycharmProjects/119NER/ui/dataicon.qrc">
    <normaloff>:/dataicon/mainIcon.png</normaloff>:/dataicon/mainIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #F8F8FF;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼매직체</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #AF1B3F;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;ASR-NER Pipeline&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="fileButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>420</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼매직체</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: #AF1B3F;</string>
    </property>
    <property name="text">
     <string>파일 선택</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recognitionButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>420</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼매직체</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: #AF1B3F;</string>
    </property>
    <property name="text">
     <string>개체명 인식 결과</string>
    </property>
   </widget>
   <widget class="QPushButton" name="maskingButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>420</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼매직체</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: #AF1B3F;</string>
    </property>
    <property name="text">
     <string>마스킹 결과</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mic">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>341</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>배달의민족 주아</family>
      <pointsize>11</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url('ui/mainIcon.png');
background-color: #F8F8FF;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../../PycharmProjects/119NER/ui/dataicon.qrc"/>
 </resources>
 <connections/>
</ui>
